drivers/hyperv/hyperv.go

```diff
@@ -428,8 +428,16 @@ func (d *Driver) generateDiskImage() (string, error) {
 	diskImage := d.ResolveStorePath("disk.vhd")
 	fixed := d.ResolveStorePath("fixed.vhd")
 
+	// Resizing vhds requires administrator priviledges
+	// incase the user is only a hyper-v admin then create the disk at the target size to avoid resizing.
+	isWindowsAdmin, err := isWindowsAdministrator()
```

Comment:
it doesn't but you won't have got this far if the function would fail - as it is run earlier in the are you admin or hyper-v admin check. 

Also I don't really get how multiple return values and assignments work - as this was a drive by submission as I wanted a working docker-machine on my machine I did not stop to attend the go-102 (I did part of the go-101 course).